test/bdd/pkg/kms/kms_steps.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package kms
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"net/url"
+	"path"
+	"strings"
+
+	"github.com/cucumber/godog"
+
+	"github.com/trustbloc/hub-kms/test/bdd/pkg/context"
+)
+
+const (
+	createKeystoreReq = `{
+	  "controller": "did:example:123456789"
+	}`
+
+	createKeyReq = `{
+	  "keystoreID": "%s",
+	  "keyType": "%s",
+	  "passphrase": "p@ssphrase"
+	}`
+
+	createKeystoreEndpoint = "{serverEndpoint}/kms/keystores"
+
+	contentType    = "application/json"
+	locationHeader = "Location"
+)
+
+// Steps defines steps context for the KMS operations.
+type Steps struct {
+	bddContext       *context.BDDContext
+	keystoreEndpoint string
+	responseStatus   int
+	responseLocation string
+}
+
+// NewSteps creates steps context for the KMS operations.
+func NewSteps() *Steps {
+	return &Steps{}
+}
+
+// SetContext sets a fresh context for every scenario.
+func (s *Steps) SetContext(ctx *context.BDDContext) {
+	s.bddContext = ctx
+}
+
+// RegisterSteps defines scenario steps.
+func (s *Steps) RegisterSteps(gs *godog.Suite) {
+	gs.Step(`^User has created a keystore on the server$`, s.createKeystore)
+	gs.Step(`^User sends an HTTP POST to "([^"]*)" to create a key of "([^"]*)" type$`, s.sendCreateKeyReq)
+	gs.Step(`^User gets a response with HTTP 201 Created and Location with a valid URL for the newly created key$`, s.checkResponse) //nolint:lll
```

Comment:
you cannot break the line to the next lint instead of disabling the linter ?

Reply:
You can if you switch it to `"`

Reply:
done with `"` and new line, thanks guys